tools/flang2/flang2exe/ll_write.cpp

```diff
@@ -400,6 +400,25 @@ ll_write_instruction(FILE *out, LL_Instruction *inst, LL_Module *module, int no_
   print_branch_target = 0;
   opname = get_op_name(inst->op);
   switch (inst->op) {
+  case LL_ASM: {
+    if(inst->num_operands==2) {
+      fprintf(out, "%scall void asm sideeffect \"%s\", \"\"()", SPACES,
+            inst->operands[1]->data);
+    } else {
+      int noperands = inst->num_operands;
+      fprintf(out, "%s%s = call %s asm sideeffect \"%s\", \"%s\"", SPACES,
+            inst->operands[0]->data, inst->operands[0]->type_struct->str, 
+            inst->operands[1]->data, inst->operands[2]->data);
+      fprintf(out, "(");
+      for(i=3; i<noperands; i++) {
+        fprintf(out, "%s %s", inst->operands[i]->type_struct->str, inst->operands[i]->data);
+        if(i<(noperands-1))
+          fprintf(out, ",");
+      }
+      fprintf(out, ")");
+    }
+  }
+  break;
```

Comment:
This should not be needed in Flang.

Reply:
Thanks, Eric. I don't necessarily have a good way of knowing what changes to the bridge are needed or unneeded in Flang. Let's talk about how I can identify that so we can limit the changes to Flang.